src/ui/Button.tsx

```diff
@@ -12,25 +14,26 @@ const StyledButton = styled.button<{
   justify-content: center;
   align-items: center;
   display: inline-flex;
+  margin: 0 0 var(--spacing-medium) 0;
```

Comment:
I think margin should be specified by the parent container, instead of a built-in attribute to Button itself.